commune/settings.py

```diff
@@ -39,6 +39,7 @@
     'django.contrib.staticfiles',
     'commune_app.apps.CommuneAppConfig',
 ]
+AUTH_USER_MODEL = 'commune_app.User'
```

Comment:
This code already merged as part of:
https://github.com/redhat-beyond/commune/pull/27

please rebase

Reply:
done.